### ISSUE TITLE
Top-level dir.props claims Roslyn can't build PDBs on Unix

### ISSUE THREAD

**Issue by SamB** [dotnet/corefx#4963]
In particular, this piece of (Unix-only) code:

```xml
    <!--
      PDB support isn't implemented yet. https://github.com/dotnet/roslyn/issues/2449
      Note that both DebugSymbols and DebugType need set or project references will assume they need to copy pdbs and fail.
    -->
    <DebugSymbols>false</DebugSymbols>
    <DebugType>none</DebugType>
```
... claims that it's necessary to disable PDBs when building on Unix because dotnet/roslyn#2449 wasn't yet ready.  However, that's been implemented, so perhaps this doesn't need to stay disabled anymore?

Also note this, just a bit further down in the same Unix-only block:
```xml
    <!--
      Building packages fails for two reasons.
      First, nuget doesn't like the paths in the nuspec having backslashes as directory separators.
      Second, we aren't yet building pdbs, which the nuspecs specify.
    -->
    <SkipBuildPackages>true</SkipBuildPackages>
```
(And, um, while we're looking at this comment: Windows is the one that uses backslashes as directory separators, not Unix, and you don't seem to have any nuspec files in this repository, so I'm not sure where any unexpected backslashes could be creeping in from.)

**Comment by weshaggard** [comment]
@joperezr please have a look.

**Comment by jasonwilliams200OK** [comment]
> Windows is the one that uses backslashes as directory separators

True, but i think msbuild accepts backslashes on Unix as well for compatibility.

**Comment by jasonwilliams200OK** [comment]
This was addressed by #5911.
@joperezr, @SamB, should this issue be closed?

**Comment by joperezr** [comment]
Yes, this is fixed, so I'm closing now. @SamB feel free to re-open if needed.
